build/configs/artik053/tc/defconfig

```diff
@@ -1324,8 +1324,8 @@ CONFIG_ENABLE_ENV_SET=y
 CONFIG_ENABLE_ENV_UNSET=y
 CONFIG_ENABLE_FREE=y
 CONFIG_ENABLE_HEAPINFO=y
-CONFIG_HEAPINFO_GROUP=y
-CONFIG_HEAPINFO_PER_GROUP="heapinfo_task,heapinfo_task2/heapinfo_task3/heapinfo_pthread"
+CONFIG_HEAPINFO_USER_GROUP=y
+CONFIG_HEAPINFO_PER_USER_GROUP="heapinfo_task,heapinfo_task2/heapinfo_task3/heapinfo_pthread"
```

Comment:
What's the meaning of "PER_USER_GROUP"?
Name should have the meaning which user can guess operation of config by name.

In logic side, first config (USER_GROUP) means enabling this feature.
second means list of group.

Could you rename them?